### ISSUE TITLE
display jpg image doesn't work

### ISSUE THREAD

**Issue by mickeydonkey** [jupyter/qtconsole#157]
```
from IPython.core.display import *
Image(data='https://upload.wikimedia.org/wikipedia/commons/thumb/2/29/Pomegranate03_edit.jpg/320px-Pomegranate03_edit.jpg', embed=True,format='jpeg')

Out[16]: <IPython.core.display.Image object>
```
qtconsole=4.2.1
python=3.5
verified on both windows and Ubuntu 16.04

@ivanov further tested in python2, seem it is relevant to python3 only.

**Comment by mickeydonkey** [comment]
Since QtConsole already support matplotlib with SVG as backend, I believe It's a good idea to support SVG display.

**Comment by takluyver** [comment]
#158 fixes it for me

**Comment by mickeydonkey** [comment]
@takluyver Very fast fix, Thank you! what's your opinion about SVG display?

**Comment by mickeydonkey** [comment]
`Image('/path/to/pattern.svg')` gives an error `Cannot embed the 'svg' image format` 
I've managed to display SVG in qtconsole, but It seems not the first-class citizen like png and jpeg.
```
svg = open('/home/luot/Pictures/pattern.svg', 'r').read()
display_svg(svg, raw=True)
```

**Comment by mickeydonkey** [comment]
seems https://github.com/ipython/ipython/issues/10043 is the right place for SVG display support.

**Comment by dashesy** [comment]
I have this problem in `Jupyter QtConsole 4.2.1` and was wondering if it is fixed in QtConsole 4.3

**Comment by wmvanvliet** [comment]
it should be. #158 was merged before the 4.3 release.
